inlong-sort/sort-connectors/sqlserver-cdc/pom.xml

```diff
@@ -36,6 +36,11 @@
             <groupId>com.ververica</groupId>
             <artifactId>flink-connector-sqlserver-cdc</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.apache.inlong</groupId>
+            <artifactId>sort-connector-base</artifactId>
+            <version>${project.version}</version>
+        </dependency>
```

Comment:
It need add `<include>org.apache.inlong:*</include>` in  maven package config. Because now packaging will lost class file of connector-base.

Reply:
done